### DISCUSSION TITLE
How to setup a Workload Identity token access on AKS to allow keycloak access on Azure?

### DISCUSSION THREAD

**Discussion by GerardoNexplore** [keycloak/keycloak#20484] [Ideas]
I wan to implement Kubernetes Workload Identity with AKS:

https://blog.baeke.info/2022/01/31/kubernetes-workload-identity-with-aks

any ideas,?

Regards
Gerardo

**Comment by Daveography** [comment]
@GerardoNexplore See https://github.com/keycloak/keycloak/discussions/26267#discussioncomment-10824627
